domain/src/test/java/uk/gov/hmcts/ccd/definition/store/domain/validation/event/EventEntityPostStateValidatorTest.java

```diff
@@ -176,7 +181,7 @@ public void successfullyValidatesPostStateConditionForCustomComplexField() throw
 
         assertNotNull(validationResult, "validation result should not be null");
         assertThat(validationResult.getValidationErrors().size(), is(0));
-        Assert.assertThat(validationResult.isValid(), CoreMatchers.is(true));
+        assertThat(validationResult.isValid(), CoreMatchers.is(true));
```

Comment:
Can `CoreMatchers.is()` be replaced with the `org.hamcrest.Matchers.is()` which is used on the previous line?  The implementations of both `is()` methods seem to point to the same thing and it would save having to import the CoreMatchers class (which is only used here).